.github/workflows/run_tests.yml

```
@@ -0,0 +1,39 @@
+name: build and test
+
+on:
+  push:
+  pull_request:
+    branches:
+      - '*' 
+    paths:
+      - '**UnitTests.csproj'
+
+env:
+  DOTNET_VERSION: '5.0.301' # The .NET SDK version to use
```

We have bumped dotnet version to `6.0.100` in the latest commit.

```suggestion
  DOTNET_VERSION: '6.0.100' # The .NET SDK version to use
```